xero-finance.yaml

```diff
@@ -1028,6 +1028,72 @@ paths:
                 title: InvalidRequest
                 status: 400
                 detail: Organisation xxx does not exist
+  /bankstatementsplus/statements:
+    get:
+      security:
+        - OAuth2: [finance.bankstatementsplus.read]
+      tags:
+        - Finance
+      operationId: getBankStatementAccounting
+      summary: Get Bank Statement Accounting
+      description: >-
+        For lenders that prefer using bank statement data as the source of
+        truth.  We provide a data point that will allow access to customer
+        bank statements, plus for reconciled bank transactions the matching
+        accounting, invoice and billing data as well.  As customers reconcile
+        bank statements to invoices and bills, this transaction detail will
+        provide valuable insight for lender's assessment measures. 
+      parameters:
+        - name: Xero-Tenant-Id
```

Comment:
`xero-tenant-id` is provided separately in the headers. The only params needed here are `in: query`